<launch>
   <include file="$(find ndt_feature)/launch/replay_without_clock.launch">
     <arg name="file" value="/home/han/data/automower/henrik/long_run_2016-04-07-19-10-00.bag" />
     <arg name="rate" value="4" />
   </include>

<!-- x=1.88 y=-0.22 a=0.015 -->
<!-- x=1.9 y=-0.14 a=0.03 -->
<!-- 1.860000 -0.060000 -0.005000 -->
<!-- 1.870000 0.000000 0.010000 -->
<!-- 1.870000 0.015000 0.017000 -->
<!-- 1.870000 0.013000 0.014000 -->
   <node pkg="ndt_feature" type="laser2d_extrinsic_calibration" name="laser2d_extrinsic_calibration" output="screen">
       	<param name="input_laser_topic" value="/scan" />
	<param name="tf_odo_topic" value="/base_link" />
	<param name="tf_state_topic" value="/base_link" />
	<param name="tf_world_frame" value="/odom" />
	      <param name="sensor_pose_x" value="0.0" />
	      <param name="sensor_pose_y" value="0.0" />
	      <param name="sensor_pose_th" value="0.0" />
	      <param name="nb_pairs" value="16" /> 
	      <param name="x_search_offset" value="0.05" />
	      <param name="y_search_offset" value="0.05" />
	      <param name="yaw_search_offset" value="0.01" />
	      <param name="xy_incr" value="0.001" /> 
	      <param name="yaw_incr" value="0.001" />
	</node>   
</launch>
